index.md

```diff
@@ -1329,6 +1329,81 @@ The `NOTICE` message is used similarly to [`PRIVMSG`](#privmsg-message). The dif
 
 One thing for bot authors to note is that the `NOTICE` message may be interpreted differently by various clients. Some clients highlight or interpret any `NOTICE` sent to a channel in the same way that a `PRIVMSG` with their nickname gets interpreted. This means that users may be irritated by the use of `NOTICE` messages rather than `PRIVMSG` messages by clients or bots, and they are not commonly used by client bots for this reason.
 
+## User-Based Queries
+
+### WHOIS message
+
+         Command: WHOIS
+      Parameters: [<target>] <nick>
+
+This command is used to query information about particular users.
+The server will answer this command with several numeric messages with information about the nicks, ending with [`RPL_ENDOFWHOIS`](#rplendofwhois-318).
+
+Servers MUST end their reply to `WHOIS` messages with one of these numerics:
+
+* [`ERR_NOSUCHNICK`](#errnosuchnick-401) `(401)`
+* [`ERR_NOSUCHSERVER`](#errnosuchserver-402) `(402)`
+* [`ERR_NONICKNAMEGIVEN`](#errnonicknamegiven-431) `(431)`
+* [`RPL_ENDOFWHOIS`](#rplendofwhois-318) otherwise, even if they did not send any other numeric message. This allows clients to stop waiting for new numerics.
+
+Client MUST NOT not assume all numerics messages are sent at once, so server can interleave other messages before the end of the WHOIS response.
+
+If the `<target>` parameter is specified, it SHOULD be a server name or the nick of a user. Servers SHOULD send the query to a specific server with that name, or to the server `<target>` is connected to, respectively.
+
+Typically, it is useful to client who want to know how long the user in question has been idle as only local server (i.e., the server the user is directly connected to) knows that information, while everything else is globally known.
```

Comment:
```suggestion
Typically, it is used by clients who want to know how long the user in question has been idle (as typically only the server the user is directly connected to knows that information, while everything else this command returns is globally known).
```
Makes sense to push this on the end of the `<target>`  paragraph, since this is just clarifying why that would be used.